opm/material/fluidsystems/BlackOilFluidSystem.hpp

```diff
@@ -88,7 +87,7 @@ auto getRv_(typename std::enable_if<HasMember_Rv<FluidState>::value, const Fluid
  *
  * \tparam Scalar The type used for scalar floating point values
  */
-template <class Scalar>
+template <class Scalar, class IndexTraits = Opm::BlackOilDefaultIndexTraits>
 class BlackOilFluidSystem : public BaseFluidSystem<Scalar, BlackOilFluidSystem<Scalar> >
```

Comment:
You should probably include the IndexTraits in the second template argument.

Reply:
what do you mean here? `IndexTraits` is the second template argument...

Reply:
ah, I see. fixed, thanks for the catch...